docs/projects/run-build-deploy.md

```diff
@@ -147,20 +161,90 @@ image that was set when the function was added to the project.
 
 For example:
 
-    project = mlrun.new_project(project_name, "./proj")
-    # use v1 of a pre-built image as default
-    project.set_default_image("myrepo/my-prebuilt-image:v1")
-    # set function without an image, will use the project's default image
-    project.set_function("mycode.py", "prep")
+```python
+ project = mlrun.new_project(project_name, "./proj")
+ # use v1 of a pre-built image as default
+ project.set_default_image("myrepo/my-prebuilt-image:v1")
+ # set function without an image, will use the project's default image
+ project.set_function("mycode.py", "prep")
+
+ # function will run with the "myrepo/my-prebuilt-image:v1" image
+ run1 = project.run_function("prep", params={"x": 7}, inputs={'data': data_url})
+
+ ...
+
+ # replace the default image with a newer v2
+ project.set_default_image("myrepo/my-prebuilt-image:v2")
+ # function will now run using the v2 version of the image 
+ run2 = project.run_function("prep", params={"x": 7}, inputs={'data': data_url})
+```
+
+<a id="build_config"></a>
+## Image build configuration
+
+Using the {py:meth}`~mlrun.projects.MlrunProject.set_default_image` function configures a project to use an existing 
```

Comment:
Use the {py:meth}`~mlrun.projects.MlrunProject.set_default_image` function to configure a project to use an existing 
image. And the configuration for building this default image can be contained within the project, by using the  {py:meth}`~mlrun.projects.MlrunProject.build_config` and {py:meth}`~mlrun.projects.MlrunProject.build_image` 
functions.